src/ws/login.css

```diff
@@ -489,6 +489,9 @@ label {
   margin-left: -20px;
   margin-right: -20px;
 }
+#option-group a {
+  color: inherit;
+}
```

Comment:
I kept this as-is, as IDs really don't belong in CSS. (`#option-group` is already used and is elsewhere too... optimally, we would probably change it to `.ct-option-group` for CSS... meaning that, in this case, the HTML might be a little redundant with `option-group` as both a class and ID, basically.)

In the self-contained login page though, it's not such a big deal. Avoiding IDs in CSS is mainly about specificity.